.eslintrc

```diff
@@ -12,9 +12,8 @@ env:
 # http://eslint.org/docs/rules/
 rules:
   # Possible Errors
-  comma-dangle: [2, never]
   no-cond-assign: 2
-  no-console: 0
+  no-console: 1
```

Comment:
Could be an error as you had in your Gist, but maybe only for `console.log` (so `no-console: [2, { allow: [warn, error] }]`)?